gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-all.zip
 networkTimeout=10000
```

Comment:
Temporarily build with source code, because atm the Gradle java doc comments are kind of useful to see when I develop.  The effective bloat is very small, and there are no performance or memory issues by changing from bin for a while. 

Upgrade to 8.6